setup/command/aws.go

```diff
@@ -23,14 +23,9 @@ var awsFlags = []cli.Flag{
 		Usage:  "secret key portion on an AWS key",
 		EnvVar: config.AWS_SECRET_ACCESS_KEY,
 	},
-	cli.StringFlag{
-		Name:   "aws-region",
-		Usage:  "AWS region",
-		EnvVar: config.AWS_REGION,
-	},
 }
 
-func (f *CommandFactory) newAWSProviderHelper(c *cli.Context) (*aws.Provider, error) {
+func (f *CommandFactory) newAWSProviderHelper(c *cli.Context, region string) (*aws.Provider, error) {
```

Comment:
[nit] is `region` unavailable from any other existing object we have access to?

Reply:
It would have been available through the CLI context, but my code removes that as a possibility. With it gone from CLI context, the only other place it could come from is Instance context.